src/index.html

```diff
@@ -7,8 +7,106 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <header class="header">
+    <ul class="ul">
+      <div class="img">
+        <a
+          href="#home"
+          class="a-image"
+        >
+          <img
+            src="images/logo.png"
+            alt="moyo"
+          >
+        </a>
+      </div>
+      <nav class="child">
+        <li class="text">
+          <a
+            href="#APPLE"
+            class="is-active"
+          >
+            Apple
+          </a>
+        </li>
+      </nav>
+      <nav class="child">
+        <li class="text">
+          <a
+            href="#SAMSUNG"
+            class="is-normal"
+          >
+            SAMSUNG
+          </a>
+        </li>
+      </nav>
+      <nav class="child">
+        <li class="text">
+          <a
+            href="#СМАРТФОНЫ И ТЕЛЕФОНЫ"
+            class="is-normal"
+          >
+            СМАРТФОНЫ И ТЕЛЕФОНЫ
```

Comment:
```suggestion
            Смартфоны и телефоны 
```
используй стили чтобы привести названия ссылок к верхнему регистру